app/controllers/api/v1/gc_organisations_controller.rb

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
 
 module Api::V1
   class GcOrganisationsController < Api::V1::ApiController
```

Comment:
Can someone remind me why we have `GcOrganisationsController` and not `OrganisationsController` ? Can we maybe rename it and redirect the routes ? cc @steveyken @bharat619 @swatijadhav @abulsayyad123 @shreyas098 

Unrelated to this PR, I just noticed

Reply:
Created a new ticket https://jira.crossroads.org.hk/browse/GCW-3292